jtravis/src/main/java/fr/inria/spirals/jtravis/helpers/Version.java

```diff
@@ -0,0 +1,14 @@
+package fr.inria.spirals.jtravis.helpers;
+
+public class Version {
+
+    private static Boolean versionV3 = true;
+
+    public static void setVersion(Boolean version) {
```

Comment:
Either you have a `setVersion` which can take an enum value, or you have `setV3` which takes a boolean value. I personnaly prefer the first one.